caluma/workflow/tests/test_work_item.py

```diff
@@ -143,7 +143,7 @@ def test_complete_workflow_form_work_item(
         question=table_question, document=answer.document, value=None
     )
     answer_document = answer_document_factory(answer=table_answer)
-    answer_document.document.answers.add(answer_factory())
+    answer_document.document.answers.add(answer_factory(value="2000-01-01"))
```

Comment:
As we now have hybrid fields, we have to jump through many hoops, as can be seen on this line. This has `value` set to "2000-01-01", because it would fail otherwise, if the answer_factory creates a DateAnswer.